src/test/java/edu/hm/hafner/util/StringContainsUtilsTest.java

```diff
@@ -31,4 +31,27 @@ void shouldSearchStrings() {
         assertThat(containsAnyIgnoreCase("This is a string text.", "wrong", "is")).isTrue();
         assertThat(containsAnyIgnoreCase("This is a string text.", "wrong", "IS")).isTrue();
     }
+
+    @Test
+    void shouldHandleNull2() {
```

Comment:
Hier wäre als Methodenname containsAllIgnoreCaseShouldHandleNull besser. Zahlen in Methodennamen, auch bei Tests, sollten vermieden werden.